truss/templates/triton/model/model.py

```diff
@@ -0,0 +1,58 @@
+from pathlib import Path
+
+import yaml
+from triton_model_wrapper import ModelWrapper
+
+
+class TritonPythonModel:
+    def __init__(self):
+        self._model_repository_path = None
+        self._config_path = None
+        self._config = None
+        self._model_wrapper: ModelWrapper = None
+
+    def _instantiate_model_wrapper(self, triton_config: dict):
+        """
+        Instantiates the model wrapper class as well as the user-defined model class.
+
+        Args:
+            triton_config (dict): Triton configuration dictionary. This contains information about the model repository
+                path and the model version.
+
+        Returns:
+            None
+        """
+        self._model_repository_path = (
+            Path(triton_config["model_repository"]) / triton_config["model_version"]
+        )
+        self._config_path = self._model_repository_path / "truss" / "config.yaml"
+        with open(self._config_path, encoding="utf-8") as config_file:
+            self._config = yaml.safe_load(config_file)
+        self._model_wrapper: ModelWrapper = ModelWrapper(self._config)
+        self._model_wrapper.instantiate()
+
+    def initialize(self, args: dict):
```

Comment:
maybe call `args` triton_config like it's called in the above function for consitency